docker/docker-compose.yml

```diff
@@ -30,6 +30,7 @@ services:
       --rpc-methods=Unsafe
       --alice
       --log="main"
+    restart: unless-stopped
```

Comment:
Automatically restarts stopped relay nodes

Reply:
Maybe make a different pr for this?